tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml

```
@@ -43,9 +43,9 @@ content_types:
         synonyms:
             - bio_data_4
 
-    -   label: Contact
-        term: local:contact
-        help_text: Use the contage page a person or institution that can be linked as a responsible party for data or results.
+    -   label: Communication Contact
```

I'm sorry for misleading you with incorrect suggestions, but this particular change affects the name of the actual content type, and in this case we don't want to change it. It was my mistake, but can you please change this one line back?
```suggestion
    -   label: Contact
```